planet/cli/orders.py

```diff
@@ -179,24 +179,23 @@ async def wait(ctx, order_id, delay, max_attempts, state):
 @translate_exceptions
 @coro
 @click.argument('order_id', type=click.UUID)
-@click.option('-o',
-              '--overwrite',
-              is_flag=True,
-              default=False,
-              help=('Overwrite files if they already exist.'))
-@click.option('--dest',
+@click.option('--directory',
               default='.',
-              help=('Directory to download files to.'),
+              help=('Root directory for file download'),
```

Comment:
```suggestion
              help=('Base directory for file download.'),
```

Reply:
Root directory usually refers to the root directory /. I recommend changing this to something else, perhaps "base directory". Also, for consistency, adding a period to the end of this sentence,